.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,49 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main]
+    paths-ignore:
+      - 'cypress/**'
+      - 'mocks/**' 
+  schedule:
+    - cron: '0 9 * * 2'
+
+env:
+  NODE_VERSION: 16
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    # Skip any PR created by dependabot to avoid permission issues
+    if: (github.actor != 'dependabot[bot]')
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: javascript
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Set up Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+    - name: Run build task
```

Can we add some whitespace here as with the other steps?
```suggestion
        node-version: ${{ env.NODE_VERSION }}

    - name: Run build task
```